dev/tasks/homebrew-formulae/apache-arrow.rb

```diff
@@ -16,7 +16,7 @@ class ApacheArrow < Formula
   depends_on "numpy"
   depends_on "openssl@1.1"
   depends_on "protobuf"
-  depends_on "python@3.8"
+  depends_on "python@3.9"
```

Comment:
I'm assuming homebrew should be on the latest Python? It does look like it's available now https://formulae.brew.sh/formula/python@3.9